### ISSUE TITLE
New Page Layout Picker: Implement the new design including design image previews

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#49587]
Update Page Layout Picker UI design per design specs.
If the new package [code](https://github.com/Automattic/wp-calypso/pull/49661) has been merged, work can be done there (and not in existing picker code).

Designs: p9Jlb4-27F-p2
User Acceptance Criteria: https://github.com/Automattic/wp-calypso/issues/49743

Tech Checklist:
- [x] Modal should not fill full viewport, instead center at 90vw & 90vh
   - [x] takes up full screen on mobile
   - [x] test with and without admin-bar present
   - [x] test with and without editor in full screen mode
- [x] Categories appear on the left, previews on the right
- [x] Blank page button appears on the left above the categories list
- [x] Categories appear in dropdown on mobile
- [x] a11y (e.g. keyboard focus stays within modal, escape closes modal, modal title and description are semantically available to screen readers etc.)
- [x] Copy is localised correctly
- [x] Modal works in RTL languages (categories on the right, close button and previews on the left)

Non-layout changes (e.g. removing the "Restaurant" category, new descriptive names, etc.) can be done in a separate PR. Potentially even in parallel and shipped before this one.

**Comment by p-jackson** [comment]
WIP PR #49860

**Comment by p-jackson** [comment]
Shipped in #49860
